app/controllers/apo_controller.rb

```diff
@@ -268,16 +286,19 @@ def update_desc_metadata
 
   def spreadsheet_template
     binary_string = RestClient.get(Argo::Config.urls.spreadsheet)
-    send_data(binary_string, :filename => 'spreadsheet_template.xlsx', :type => 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet')
```

Comment:
hound.ci would not pass the PR (this comment applies to all similar concerns for Jack).